libs/@local/graph/postgres-store/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -176,6 +177,15 @@ where
             for (entity_vertex_id, graph_resolve_depths, traversal_interval) in
                 entity_queue.drain(..)
             {
+                let _span = tracing::trace_span!(
```

Comment:
Could we maybe move this into an external function (or even closure) that isn't `async` or use https://docs.rs/tracing/latest/tracing/span/struct.Span.html#method.in_scope ? This is motivated by https://docs.rs/tracing/latest/tracing/span/struct.Span.html#in-asynchronous-code. I could imagine us changing this code in the future and then suddenly having rogue `traverse_edges` span in the tree.